### DISCUSSION TITLE
Why "bob"?

### DISCUSSION THREAD

**Discussion by joshpetit** [MordechaiHadad/bob#76] [General]
So, why exactly bob?

**Comment by vsedov** [comment]
The Name? Or why should one use bob? 

The question you should be asking is who is bob ?
> Bob the tomato

**Reply by joshpetit** [reply]
the name lol

**Comment by MordechaiHadad** [comment]
Basically this:
![](https://p0rn.pro/APurpleAndImportedPlace)

**Reply by joshpetit** [reply]
Sir Polo is in fact correct, you can never go wrong with the name bob.

**Comment by joshpetit** [comment]
It must be an honor to know that millions of parents worldwide chose to name their child after this project.

**Reply by MordechaiHadad** [reply]
Damn I never thought of this...
Next year on bob's first commit date if someone names their newborn bob on that day, they will get a free copy of bob just like how Bethesda did with Skyrim
